engine/src/flutter/impeller/entity/contents/content_context.h

```diff
@@ -292,6 +293,16 @@ class RenderTargetCache;
 
 class ContentContext {
  public:
+  struct Settings {
+    bool lazy_shader_mode = false;
+  };
```

Comment:
I'm working on a flags system for impeller right now.  I think we're better off with a holistic flags system for all of impeller instead of these different settings distributed across the codebase and sometimes living as parameter arguments.

I planned on migrating the wifeframe option to this system too.

It will be something like this:
```
namespace impeller {
class Flags {
  public:
    void Init(std::set<std::string_view> names);
    bool HasFlag(std::string_view name) const;
};
}
```

Reply:
slightly related, I'd actually like us to remove wireframe as its a debugging option we no longer use.


better flags SGTM tho. Barring other feedback, I'd like to land this first to unblock the customer money tests. If there are things I can change to make it work better with your flag proposal that would be fine too.

Reply:
That SGTM

Reply:
Okay, I refactored it to use impeller::Flags.  I also uncommented out that function call like we discussed in the other PR.  Let's try getting this through together, if it gets too weird I can rip out the flags stuff into a new pr and we can rebase this pr on it.